res/css/views/rooms/_EventBubbleTile.scss

```diff
@@ -263,29 +263,40 @@ limitations under the License.
             z-index: 3; // above media and location share maps
         }
 
-        &.mx_EventTile_mediaLine .mx_MVoiceMessageBody {
-            // allow the event to be collapsed, this causes the waveform to get cropped
-            min-width: 0;
-        }
+        &.mx_EventTile_mediaLine {
+            > div,
```

Comment:
If possible we should target the specific elements by classes. For the two here: `mx_MFileBody`, `mx_MAudioBody`.

I understand the idea of automagically covering all elements. On the other hand:
- If a `div`/`span` is changed to `a` or whatever the rule breaks
- If other elements are added there have to be more exceptions like for the `mx_MessageActionBar` currently
- You can find the rules affecting this elements easily in the code if you have a class name

If there should be a „catch all“ we should add some common class to all the „body“ elements and target it here.

Reply:
I agree with all of them, and I am pretty sure that they are ideally how elements should be styled. The only issue is that there is not a general style rule about how to specify all elements. Some cases bare selectors are used like this, other cases class names are used... it looks like this kind of inconsistency has partially contributed to style inconsistencies.

> If other elements are added there have to be more exceptions like for the mx_MessageActionBar currently

It is totally true, and the same thing applies to styling elements using class name. Basically you would need to maintain the list by adding a new class name, which could make the list bloated like this: https://github.com/matrix-org/matrix-react-sdk/blob/11cb48176e01befe00d81f1a119c9a76c33be9b4/res/css/views/rooms/_EventTile.scss#L910-L917

> If there should be a „catch all“ we should add some common class to all the „body“ elements and target it here.

I think this would be a proper way of styling elements, not only maintaining the style consistency, but also minimizing the manual works required for it.

Reply:
However, because it requires an analysis about how class name should added in each case, for now, I am going to specify elements with class name, hoping the list would not be messy... :crossed_fingers: